README.md

```diff
@@ -56,3 +56,17 @@ Note that repeated tests do run concurrently with themselves for efficiency, and
 as such they have problem writing to hard-coded files, even if they are only
 used by that single test. `tmpfile()` and similar library functions are often
 your friends here.
+
+## Running tests cases in parallel
+
+Sometimes tests in test case make system-wide changes in their
```

Comment:
Sometimes tests within a single test case use globally-shared resources (instead of temporary files or fake/mock objects) and cannot be run in parallel. Examples of...